src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/SupplierPricingService.kt

```diff
@@ -12,6 +13,7 @@ import uk.gov.justice.digital.hmpps.pecs.jpc.price.Supplier
 
 @Service
 @Transactional
+@PreAuthorize("hasRole('PECS_MAINTAIN_PRICE')")
```

Comment:
Same here. Using class level as all of the controller methods here required role this particular role enforcement.